frontend/src/scenes/funnels/FunnelHistogram.tsx

```diff
@@ -23,26 +25,33 @@ export function FunnelHistogramHeader(): JSX.Element | null {
             <span className="funnel__header__steps__label">Steps</span>
             {histogramStepsDropdown.length > 0 && (
                 <Select
-                    defaultValue={histogramStepsDropdown[0]?.value}
-                    onChange={changeHistogramStep}
+                    defaultValue={histogramStepsDropdown[0]?.from_step}
+                    onChange={(from_step) => {
+                        changeHistogramStep(from_step, from_step + 1)
+                    }}
                     dropdownMatchSelectWidth={false}
+                    dropdownAlign={ANTD_TOOLTIP_PLACEMENTS.bottomLeft}
                     data-attr="funnel-bar-layout-selector"
                     optionLabelProp="label"
                 >
                     {histogramStepsDropdown.map((option, i) => {
                         const basisStep = getReferenceStep(stepsWithCount, stepReference, i)
                         return (
-                            <Select.Option key={option?.value} value={option?.value || 1} label={<>{option?.label}</>}>
+                            <Select.Option
+                                key={option?.from_step}
```

Comment:
Should an `undefined` be acceptable for the `key` & `value`? Perhaps we shouldn't render an option if this is undefined?

Reply:
According to how we construct `histogramStepsDropdown` [here](https://github.com/PostHog/posthog/blob/fb52bdfad2737c0f9a4bfd1740900ef768e274f1/frontend/src/scenes/funnels/funnelLogic.ts#L339-L360), it looks like `from_step` will never be undefined. I'll remove these conditional checks here b/c I think they're unnecessary.